api/coordinator_test.go

```diff
@@ -97,6 +97,7 @@ func TestCoordinatorAPI(t *testing.T) {
 		}
 
 		data.CoordinatorConfig.ClusterConfig.Agents[0].ApiURL = server.URL
+		data.CoordinatorConfig.ClusterConfig.MaxActiveUsers = 1000
```

Comment:
We need to set this above and the value should be 100 since that's what we are using for the agent:

```patch
diff --git a/api/coordinator_test.go b/api/coordinator_test.go
index 187227d..c67f69d 100644
--- a/api/coordinator_test.go
+++ b/api/coordinator_test.go
@@ -40,6 +40,7 @@ func TestCoordinatorAPI(t *testing.T) {
        config.MonitorConfig.Queries[0].Description = "Query"
        config.MonitorConfig.Queries[0].Query = "query"
        config.ClusterConfig.Agents[0].ApiURL = server.URL
+       config.ClusterConfig.MaxActiveUsers = 100
 
        t.Run("create/destroy", func(t *testing.T) {
                data := struct {
@@ -97,7 +98,6 @@ func TestCoordinatorAPI(t *testing.T) {
                }
 
                data.CoordinatorConfig.ClusterConfig.Agents[0].ApiURL = server.URL
-               data.CoordinatorConfig.ClusterConfig.MaxActiveUsers = 1000
 
                obj := e.POST("/create").WithQuery("id", id).WithJSON(data).
                        Expect().Status(http.StatusCreated).
```

By the way you can also run `make test` locally to verify it works.